### PR TITLE
e-lostar add caption

### PR DESCRIPTION
**Pull request by e-lostar**
TL;DR Adding a new caption for testing purpose.
Related PR: #4  

## Done
- bullet01
- bullet02

## To do
- [ ] checkbox01
- [ ] checkbox02

### CODE REVIEW

File: _slides/04-e-lostar.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Başlık
```
Comment (egencer) [inline _slides/04-e-lostar.md:6]: Good caption! 💯

File: _slides/04-e-lostar.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (egencer) [inline _slides/04-e-lostar.md:3]: Great work! 👍

**Review by egencer — APPROVED** [review]
Ready to ship 🚢

File: _slides/04-e-lostar.md (hunk 1 above)
Comment (gusshawstewart) [inline _slides/04-e-lostar.md:6]: Cool! : 😎

**Review by gusshawstewart — APPROVED** [review]

**Pull request merged**